daemon/src/engine/framework/System.cpp

```diff
@@ -511,8 +511,8 @@ static void Init(int argc, char** argv)
 #ifdef _WIN32
 	// If we were launched from a console, make our output visible on it
 	if (AttachConsole(ATTACH_PARENT_PROCESS)) {
-		freopen("CONOUT$", "w", stdout);
-		freopen("CONOUT$", "w", stderr);
+		Q_UNUSED(freopen("CONOUT$", "w", stdout));
+		Q_UNUSED(freopen("CONOUT$", "w", stderr));
```

Comment:
Just to fix 'Unused return value' warning. 
From http://www.cplusplus.com/reference/cstdio/freopen/ freopen returns stream (success) or NULL (failure). 

Do you suggest to check for null return value and emit a warning, or smth else?